src/main/kotlin/org/kamiblue/client/module/modules/misc/NoteBot.kt

```diff
@@ -315,4 +323,107 @@ internal object NoteBot : Module(
             }
         }
     }
+
+    private fun sortOutInstruments() {
+        if (!isNbsFormat) return
+
+        channelSettings[0].value = NoteBlockEvent.Instrument.PIANO
+        channelSettings[1].value = NoteBlockEvent.Instrument.BASSGUITAR
+        channelSettings[2].value = NoteBlockEvent.Instrument.BASSDRUM
+        channelSettings[3].value = NoteBlockEvent.Instrument.SNARE
+        channelSettings[4].value = NoteBlockEvent.Instrument.CLICKS
+        channelSettings[5].value = NoteBlockEvent.Instrument.GUITAR
+        channelSettings[6].value = NoteBlockEvent.Instrument.FLUTE
+        channelSettings[7].value = NoteBlockEvent.Instrument.BELL
+        channelSettings[8].value = NoteBlockEvent.Instrument.CHIME
+        channelSettings[9].value = NoteBlockEvent.Instrument.XYLOPHONE
+    }
+
+    private fun readNbs(fileName: String): TreeMap<Long, ArrayList<Note>> {
+        val noteSequence = TreeMap<Long, ArrayList<Note>>()
+        val file = File(fileName)
+        val dataInputStream = DataInputStream(FileInputStream(file))
+        var length = dataInputStream.readShort()
+
+        var nbsVersion = 0
+        if (length.toInt() == 0) {
+            nbsVersion = dataInputStream.readByte().toInt()
+            dataInputStream.readByte().toInt()
+
+
+            if (nbsVersion >= 3) {
+                @Suppress("UNUSED_VALUE")
```

Comment:
I'll merge once I can put a comment as to why length is unused

Reply:
Just remove the value assign if you just want to skip the values